zcash_client_sqlite/src/wallet.rs

```diff
@@ -929,8 +921,7 @@ pub(crate) fn get_unspent_transparent_outputs<P: consensus::Parameters>(
     max_height: BlockHeight,
 ) -> Result<Vec<WalletTransparentOutput>, SqliteClientError> {
     let mut stmt_blocks = wdb.conn.prepare(
-        "SELECT u.received_by_account,
```

Comment:
To clarify, we were using the same `WalletTransparentOutput` struct to both load UTXOs into the database, and return them to the caller of this method? Does the caller of this method not require knowledge of the account?

Reply:
The only thing we're doing with the results of this method is computing the transparent balance, so all we use is actually just the `value` field. When we go to implement the ZIP 315 recommendations, we'll probably want to add methods to the data access api that give this information more directly; right now, all the lookups are done by address rather than account and so it's a bit of a mess in any case.